### PR TITLE
Template initial microgrid subsystem example

### PR DESCRIPTION
**Pull request by vtnate**
#### Any background context you want to provide?
Multiple electrical components are required to make a single microgrid. This is the next step towards a fully functioning level 3 microgrid model, pulling together individual custom components into a system.

#### What does this PR accomplish?
Pulls "level 1" parameters from a sys-param file and populates a subsystem template
#### How should this be manually tested?
Automated test `tests/GMT_Lib/test_gmt_lib.py::test_simulate_pv_subsystem` should be sufficient.

### CODE REVIEW

**Comment by vtnate** [comment]
Sure wish I knew why Poetry has failures installing dependencies sometimes but not always! 🤷 I haven't seen it happen on the Linux runners since changing Actions, and it doesn't happen as often as before. It took until the 11th re-run for this most recent commit to succeed 😬 😢 😡

File: geojson_modelica_translator/system_parameters/system_parameters.py [hunk 1]
```diff
@@ -480,8 +479,8 @@ def process_electrical_components(self, scenario_dir: Path):
             capacitor banks (todo)
         """
         dss_data = {}
-        opendss_json_file = os.path.join(scenario_dir, 'scenario_report_opendss.json')
-        if (os.path.exists(opendss_json_file)):
+        opendss_json_file = Path(scenario_dir) / 'scenario_report_opendss.json'
```
Comment (nllong) [inline geojson_modelica_translator/system_parameters/system_parameters.py:482]: you ❤️ replacing these :)

File: tests/GMT_Lib/test_gmt_lib.py [hunk 2]
```diff
@@ -160,6 +170,7 @@ def test_simulate_community_pv():
     assert success is True
 
 
+@pytest.mark.skip(reason="This functionality is entirely captured by test_simulate_wind_turbine")
```
Comment (nllong) [inline tests/GMT_Lib/test_gmt_lib.py:173]: any reason to not just delete?

File: tests/model_connectors/test_district_heating_and_cooling_systems.py [hunk 3]
```diff
@@ -107,12 +107,15 @@ def test_build_district_heating_and_cooling_systems(self):
     #                                       project_name=self.district._scaffold.project_name)
 
     @pytest.mark.simulation
-    @pytest.mark.skip("OMC Failed. Simulation completed, but lots of errors in stdout.log")
+    @pytest.mark.skip("Succeeds when run manually")
```
Comment (nllong) [inline tests/model_connectors/test_district_heating_and_cooling_systems.py:110]: oh my

**Review by nllong — APPROVED** [review]

File: geojson_modelica_translator/system_parameters/system_parameters.py (hunk 1 above)
Comment (vtnate) [reply geojson_modelica_translator/system_parameters/system_parameters.py:482]: 😆 yeah... 🤷 

Call it modernizing to feel better about my neuroses? 😆

File: tests/GMT_Lib/test_gmt_lib.py (hunk 2 above)
Comment (vtnate) [reply tests/GMT_Lib/test_gmt_lib.py:173]: Once/if I implement https://github.com/urbanopt/geojson-modelica-translator/issues/582 then it'll be nice to have them separated as they currently are.

File: tests/model_connectors/test_district_heating_and_cooling_systems.py (hunk 3 above)
Comment (vtnate) [reply tests/model_connectors/test_district_heating_and_cooling_systems.py:110]: Gah. There was a typo in one of the assert statements that I didn't catch earlier. This test shouldn't be skipped. Fixing now.

**Pull request merged**